src/styles/blocks/card.scss

```diff
@@ -0,0 +1,63 @@
+.card {
+  width: 200px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  border-radius: 5px;
+  padding-top: 32px;
+  border: 1px solid #f3f3f3;
+  background-color: #fff;
+
+  &__image {
+    display: block;
+    width: 160px;
+    height: 134px;
+  }
+
+  &__title-wrapper {
+    margin-top: 40px;
+    text-align: left;
+  }
+
+  &__title {
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    color: $color-tertiary;
+    margin: 0 16px 4px;
+  }
+
+  &__subtitle {
+    font-size: 10px;
+    line-height: 14px;
+    font-weight: 400;
+    color: $color-secondary;
+    text-align: left;
+    margin: 4px 16px 16px;
+  }
+
+  &__buybtn {
+    height: 40px;
+    width: calc(100% - 34px);
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    background-color: $color-primary;
```

Comment:
Ensure that the variable `$color-primary` is defined in your SCSS files to avoid compilation errors.